### DISCUSSION TITLE
내가 기록하는 이유 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#56] [Announcements]
# 내가 기록하는 이유 - 정현수 기술 블로그

글또 9기를 시작하면서

https://junghyeonsu.com/posts/geultto-9th-commitment/

**Comment by JhsK** [comment]
안녕하세요! 평소에 글 너무 잘 보고 있습니다! 혹시 글을 작성할 때 썸네일 이미지는 어떻게 만드시나요?!

**Reply by junghyeonsu** [reply]
글 읽어주셔서 감사합니다!
썸네일은 제 개인 피그마에서 템플릿을 만들어놓고 직접 작업해서 올립니다ㅎㅎ
기술 포스팅 썸네일 같은 경우는 예전엔 [DallE](https://labs.openai.com/) 이용해서 만들었었어요.

**Reply by JhsK** [reply]
아하 직접 작업하시는군요! DallE도 참고해보겠습니다. 감사합니다!!
새로운 포스트도 기대하고 있겠습니다!
